rlclientlib/logger/async_batcher.h

```diff
@@ -153,9 +164,37 @@ namespace reinforcement_learning { namespace logger {
 
     RETURN_IF_FAIL(collection_serializer.finalize(status));
 
+    if (_events_counter_status == events_counter_status::ENABLE) {
+      _buffer_end_event_number = evt.get_event_number();
```

Comment:
Can we change _buffer_start_event_number from a member variable to a local variable as it is only used in this method?

Reply:
changed